### ISSUE TITLE
hidden footer links when zoomed or custom text spacing applied (overflow hidden)

### ISSUE THREAD

**Issue by notlee** [Financial-Times/origami#931]
>Some users rely on text spacing options when navigating a page to read and access content. In doing so, the user expects all content to remain accessible and available, without it being overlapped or spilling off the page. However, when a user does navigate with text spacing options, the content within the footer such as the link ‘Slavery Statement & Policies’ is not all accessible for the user, as the links do not wrap, meaning the user does not have access to all of the link text to clearly understand the links purpose and destination.
>This is due to the content being marked up with a ‘nowrap’ within the CSS.

https://financialtimes.atlassian.net/browse/CON-2092
